src/my_assistant.py

```diff
@@ -33,8 +41,11 @@
 
 # Presa in input una stringa, questa verrà esposta a voce
 def speak(response: str) -> None:
-    engine.say(response)
-    engine.runAndWait()
+    if sys.platform == 'Linux' or sys.platform == 'win32':
```

Comment:
Scusami per il suggerimento, infatti pylint suggerisce di effettuare il confronto utilizzando un set quindi: 
```suggestion
    if sys.platform in ('Linux' , 'win32'):
```

Reply:
done ✅